tests/fixtures/blueprints/queues.json

```diff
@@ -3,67 +3,67 @@
         "FifoArn": {
             "Value": {
                 "Fn::GetAtt": [
-                    "Fifo",
+                    "Fifo", 
                     "Arn"
                 ]
             }
-        },
+        }, 
         "FifoUrl": {
             "Value": {
                 "Ref": "Fifo"
             }
-        },
+        }, 
         "RedrivePolicyArn": {
             "Value": {
                 "Fn::GetAtt": [
-                    "RedrivePolicy",
+                    "RedrivePolicy", 
                     "Arn"
                 ]
             }
-        },
+        }, 
         "RedrivePolicyUrl": {
             "Value": {
                 "Ref": "RedrivePolicy"
             }
-        },
+        }, 
         "SimpleArn": {
             "Value": {
                 "Fn::GetAtt": [
-                    "Simple",
+                    "Simple", 
                     "Arn"
                 ]
             }
-        },
+        }, 
         "SimpleUrl": {
             "Value": {
                 "Ref": "Simple"
             }
         }
-    },
+    }, 
     "Resources": {
         "Fifo": {
             "Properties": {
-                "FifoQueue": true,
+                "FifoQueue": true, 
                 "QueueName": "Fifo.fifo"
-            },
+            }, 
```

Comment:
Any idea where all these whitespace changes are coming from?

Reply:
No idea. It happened to @russellballestrini as well a while back: https://github.com/remind101/stacker_blueprints/pull/163